### PR TITLE
Display suggestions when importing seed phrase #1568

### LINKED ISSUE #1568: Display suggestions when importing seed phrase
Please use Moonlight app as a reference. 
Please improve the look of segmented controls. 

Import_with_Seed_iOS

### PR DESCRIPTION
**Pull request by nimitparekh2020**
Fixes #1568

### CODE REVIEW

File: AlphaWallet/Wallet/ViewControllers/suggested_keyword.plist [hunk 1]
```diff
@@ -0,0 +1,2054 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<array>
+    <string>abandon</string>
```
Comment (hboon) [inline AlphaWallet/Wallet/ViewControllers/suggested_keyword.plist:5]: @nimitparekh2020 will review. But a quick feedback. This file doesn't need to be added. The word list is available as `HDWallet.englishWordList`.

Comment (nimitparekh2020) [reply AlphaWallet/Wallet/ViewControllers/suggested_keyword.plist:5]: I will create new function into `HDWallet` class, which returns all `mnemonic` then It fetches into into `ViewController` and I will remove created `.plist`.

**Comment by nimitparekh2020** [comment]
I developed like android functionality once the first word insert not gives any suggestion than from the second word insert more than 1 character it shows suggestions like below output shows.

**Comment by hboon** [comment]
Superseded by #1821

**Pull request closed without merging**